Library/ViewModels/PledgeViewModel.swift

```diff
@@ -482,6 +482,8 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .switchMap { [checkoutIdProperty] input in
         AppEnvironment.current.apiService.createBacking(input: input)
           .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          // FIXME: Remove once UpdateBacking mutation is aligned on error type.
```

Comment:
This was only needed in `PledgeViewModel` because this is later merged with the `updateBacking` signal and they would now have dissimilar error types. Once I migrate the update backing mutation this can be removed.